samples/Azure.Management.Storage/Generated/Extensions/SubscriptionExtensions.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using Azure.ResourceManager.Core;
+
+namespace Azure.Management.Storage
+{
+    /// <summary> Extension methods for convenient access on SubscriptionOperations in a client. </summary>
+    public static partial class SubscriptionExtensions
+    {
+        #region Sku
+        /// <summary> Gets an object representing a SkuContainer along with the instance operations that can be performed on it. </summary>
+        /// <param name="subscription"> The <see cref="SubscriptionOperations" /> instance the method will execute against. </param>
+        public static SkuContainer GetSkuContainer(this SubscriptionOperations subscription)
+        {
+            throw new NotImplementedException();
```

Comment:
Container class ctor is not implemented yet, so throwing an exception for now. After the container class gets implemented we can uncomment [this](https://github.com/Azure/autorest.csharp/blob/dd0693e10b405aff1362fda8f706e4f6e56b7fa7/src/AutoRest.CSharp/Mgmt/Generation/SubscriptionExtensionsWriter.cs#L93-L94) code.

Reply:
Based on this comment I wouldn't expect any changes to all of the Container files.

My suggestion would be to change the container constructor to match the proto which only takes in the operationsBase but no data.  Then from here you simply pass in subscription for that parameter